c++/src/kj/memory.h

```diff
@@ -22,6 +22,8 @@
 #pragma once
 
 #include "common.h"
+#include <atomic>
+#include <cstddef>
```

Comment:
```suggestion
#include <stddef.h>
```

What makes this header needed, exactly?

Reply:
this is not needed anymore, you're right (uint64_t)